book/visual-effects.md

```diff
@@ -413,158 +421,155 @@ and macOS, and Skia in Android.
 Pixels, Color, Raster
 =====================
 
-Now that we've gotten code upgrades out of the way, it's time to learn about
-how reasterization works.
-
-Rasterization---turning display lists into pixels---is the main sub-task of
-the "draw" rendering step. Skia^[And Tkinter also, of course.] does raster for
-us. But so far we just called APIs on those libraries, and didn't really
-dig into what is going on at any deeper level. Let's start to do that
-now.
-
-As you probably already know, computer screens are a 2D array of pixels. Each
-pixel has a color, which looks that way to a human because it emits a mix of
-light at different *color channel* frequencies. For example, there could be a
-red, green and blue light embedded within the physical pixel at a frequency
-closely matching the light-detecting [cones][cones] in a human eye, and each
-color could be set to any intensity.[^human-color] And as you learned in
-physics class, adding together these colors results in a combined color that,
-for different values of red, green and blue, looks like most any color a human
-can see. The three colors and how they mix define what is called a *color
-space*, and the set of colors you can express from them is called its *gamut*.
-
-[^human-color]: We all take color for granted in our lives. But just as computer
-screens simulate colors humans happen to be able to see, the colors we can see
-are not random at all, and have to do with the frequencies of light emitted by
-the sun, which our brains interpret as color. Color, and human perception of
-it, is a very interesting topic.
-
-[cones]: https://en.wikipedia.org/wiki/Cone_cell
-
-Red, green and blue was the approach taken in many computer screens, in
-particular monitors from the 80s and 90s using [CRT] technology. Since that was
-when the web came into existence, CSS uses a [sRGB] color space derived from
-and calibrated for this technology.[^other-spaces] In thos color space, each of
-the three color channels (red, green and blue) have a floating-point value
-between 0 and 1, with 0 being "off" 1 being "as bright as possible".
-
+Skia, like the Tkinter canvas we've been using until now, is a
+_rasterization_ library: it converts shapes like rectangles and text
+into pixels. Before we move on to Skia's advanced features, let's look
+at how rasterization works at a deeper level.
+
+You probably already know that computer screens are a 2D array of
+pixels. Each pixel contains red, green and blue lights,[^lcd-design]
+or _color channels_, that can shine with an intensity between 0 (off)
+and 1 (fully on). By mixing red, green, and blue, any color can be
+made. More precisely, the set of colors that can be made this way is
+called the [sRGB color space][srgb], and web pages use those colors
+via CSS.[^other-spaces]
+
+[^lcd-design]: Actually, some screens contain [pixels besides red,
```

Comment:
Add a sentence at the beginning, porting over from my text:

"sRGB is a color space defined in the 90s based on [CRT]-technology screens of the time. Since that was
when the web came into existence, CSS used this color space."

Reply:
I put this into the "other-spaces" footnote just below. Do you think it should be moved to this one?

Reply:
Oh right. I found that later in my code review and forgot to delete this comment. It's ok where it is now.